.github/workflows/rebuildDependencies.yml

```diff
@@ -134,20 +135,28 @@ jobs:
         git sparse-checkout set \
           recipes/minizip \
           recipes/flac \
+          recipes/onnx \
+          recipes/onnxruntime \
           $recipePathQt \
 
         git checkout
 
+        # Patch conan recipe to prevent dragging libcurl as a dependency
+        # (can't be done via VCMI's conanfile as onnxruntime is built beforehand)
+        git apply --ignore-whitespace ../conan_patches/onnxruntime/recipe.diff
```

Comment:
you can pass `date/*:use_system_tz_db` as an option to `conan create` or simply place it in our profile, to `common` I guess

Reply:
The conan recipe also contains this change:
```diff
     def _compilers_minimum_version(self):
         return {
             "Visual Studio": "17",
-            "msvc": "193",
+            "msvc": "192",
             "gcc": "9",
```

It can't be modified a command-line options AFAIK, which means the patch remains anyway. Does it still make sense to extract the `date/*:use_system_tz_db` into a cmdline option and wouldn't that make it harder to follow?

Reply:
true, this diff must stay as a patch.

I'd make patches as small as possible to make less changes in the future whenever patches will stop applying cleanly.

you could also place `date/*:use_system_tz_db` to our conanfile.py as it's basically a hard requirement if I understand correctly.